src/Tasks/Common/Resources/Strings.resx

```diff
@@ -525,13 +525,21 @@ The following are names of parameters or literal values and should not be transl
     <value>NETSDK1105: Windows desktop applications are only supported on .NET Core 3.0 or higher.</value>
     <comment>{StrBegin="NETSDK1105: "}</comment>
   </data>
+  <data name="WindowsDesktopFrameworkRequiresUseWpfOrUseWindowsForms" xml:space="preserve">
+    <value>NETSDK1106: Microsoft.NET.Sdk.WindowsDesktop requires 'UseWpf' or 'UseWindowsForms' to be set to 'true'</value>
+    <comment>{StrBegin="NETSDK1106: "}</comment>
+  </data>
   <data name="UseWpfOrUseWindowsFormsRequiresWindowsDesktopFramework" xml:space="preserve">
     <value>NETSDK1107: Microsoft.NET.Sdk.WindowsDesktop is required to build Windows desktop applications. 'UseWpf' and 'UseWindowsForms' are not supported by the current SDK.</value>
     <comment>{StrBegin="NETSDK1107: "}</comment>
   </data>
-  <data name="WindowsDesktopFrameworkRequiresUseWpfOrUseWindowsForms" xml:space="preserve">
-    <value>NETSDK1106: Microsoft.NET.Sdk.WindowsDesktop requires 'UseWpf' or 'UseWindowsForms' to be set to 'true'</value>
-    <comment>{StrBegin="NETSDK1106: "}</comment>
+  <data name="RuntimeListNotFound" xml:space="preserve">
+    <value>NETSDK1108: Runtime list file '{0}' was not found. Report this error to the .NET Core team here: https://github.com/dotnet/core-setup/issues.</value>
```

Comment:
I'm definitely open to suggestions regarding these two error messages.

Reply:
I think there was some discussion in the past about whether we wanted to have links to our repos for filing issues in our error messages. @nguerrera

Reply:
This is primarily why I wanted suggestions 😄.  These are internal errors that are not actionable by users directly and I'd love to point them towards a way to inform us of the problem, even if it's really unlikely users would ever run into these before we do.  But I can understand not wanting repo references in the error messages.

Reply:
We could make an aka.ms/report-runtime-list-issue or something?

I don't actually recall such a discussion at this point. cc @livarcocc

Reply:
Maybe a more generic `aka.ms/dotnet-core-issue` that takes them to `dotnet/core` and we can triage from there?

Reply:
That works for me.

Reply:
What about using `aka.ms/dotnet-core-sdk-issue` and we can point it at either the CLI or SDK repos? It's likely that the issue really belongs to core-setup for these two errors, but I think we should be responsible for triage given the errors are coming from the SDK.

We can then use the aka.ms link for any other internal SDK errors in the future.

Reply:
Ship it.